drivers/esci/Makefile.am

```diff
@@ -37,7 +37,7 @@ driverlibdir    = $(pkglibdir)
 driverdatadir   = $(pkgdatadir)/drivers/esci
 driver_ldflags  = $(AM_LDFLAGS)
 driver_ldflags += -export-dynamic
-driver_ldflags += -export-symbols-regex libdrv_$(driver)_LTX_factory
```

Comment:
Nice catch!  Will fix this together with the other instance in a single commit attributing you.